### ISSUE TITLE
Windows support?

### ISSUE THREAD

**Issue by ruboerner** [allefeld/mkernel#1]
Thanks for developing this kernel. I have used it under MacOS without problems. However, when running
`quarto preview mkernel.qmd`
on Windows, I get the following error messsage:

```
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\rub\miniconda3\Lib\site-packages\mkernel\__main__.py", line 12, in <module>
    from .kernel import MKernel
  File "C:\Users\rub\miniconda3\Lib\site-packages\mkernel\kernel.py", line 20, in <module>
    from wurlitzer import pipes
  File "C:\Users\rub\miniconda3\Lib\site-packages\wurlitzer.py", line 30, in <module>
    from fcntl import F_GETFL, F_SETFL, fcntl
ModuleNotFoundError: No module named 'fcntl'
```

It seems that `fcntl` is not supported in Windows OS?

**Comment by allefeld** [comment]
That error comes from importing `wurlitzer`, and they have an open issue regarding that: https://github.com/minrk/wurlitzer/issues/12. It's from 2018, so it seems it's not easy to resolve.

Wurlitzer is used to implement the "wrapper" method for output capture, which isn't used with Quarto by default anyway.

I just pushed 59dd311, which should catch the error and disable the "wrapper" method. Could you try this version and let me know whether it fixes the issue?

**Comment by ruboerner** [comment]
Thanks for the update, mkernel seems to be fixed now!

![Screenshot 2023-11-21 074208](https://github.com/allefeld/mkernel/assets/6057466/8bf6e613-910d-4dde-a943-726d29812433)

**Comment by allefeld** [comment]
Fix released with [v1.0.1](https://github.com/allefeld/mkernel/releases/tag/v1.0.1).
